src/systems/painter.js

```diff
@@ -73,6 +74,13 @@ AFRAME.registerSystem('painter', {
       if (event.keyCode === 86) { // v - save
         self.save();
       }
+      if (event.keyCode === 79){
```

Comment:
Can you comment which key is it and it function? like `// Y - whatever`
Also you need a space between `){`